src/services/config.ts

```diff
@@ -23,6 +24,7 @@ export default class Config {
   readonly YOUTUBE_API_KEY!: string;
   readonly SPOTIFY_CLIENT_ID!: string;
   readonly SPOTIFY_CLIENT_SECRET!: string;
+  readonly NODE_ENV!: string;
```

Comment:
We only have two modes, development and production. I think it might make more sense to have `IS_PRODUCTION = process.env.NODE_ENV === 'production'` here instead, so we don't have to manually compare the value with a string elsewhere.